### PR TITLE
add scikit-video (to replace sk-video)

### PR DESCRIPTION
**Pull request by carlodri**
adding this recipe to replace the existing sk-video feedstock (copy&pasting from https://github.com/conda-forge/sk-video-feedstock/pull/7).

sk-video feedstock needs to be discontinued/archived.

### CODE REVIEW

File: recipes/scikit-video/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
```
Comment (ocefpaf) [inline recipes/scikit-video/meta.yaml]: It looks like this builds against `numpy`, if so please take a look at:

https://conda-forge.org/docs/meta.html#building-against-numpy

File: recipes/scikit-video/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{% set name = "scikit-video" %}
+{% set version = "1.1.11" %}
+{% set sha256 = "5061d2aeae1892b73a97c89a82942b3e8eebf2fe23e59c60e06ede5f8a24ed1e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/scikit-video/meta.yaml]: You can remove this line. Latest `conda-build` can use the URL to get the name.

File: recipes/scikit-video/meta.yaml (hunk 1 above)
Comment (carlodri) [reply recipes/scikit-video/meta.yaml]: I'm not sure what "links against numpy" means exactly, but looking at https://github.com/scikit-video/scikit-video/blob/master/setup.py I only see a `from numpy.distutils.core import setup`, which from the docs doesn't seem to be a "link" case. What do you think?

Comment (ocefpaf) [reply recipes/scikit-video/meta.yaml]: It is creating a compiled extension, right? If so it is probably safer to use the "building against numpy" instructions.

Comment (ocefpaf) [reply recipes/scikit-video/meta.yaml]: That is a common things when building `numpy` extensions if I remember correctly the `numpy` import monkey patches the `setuptools`.

Comment (carlodri) [reply recipes/scikit-video/meta.yaml]: 😕 doesn't seem so: https://github.com/numpy/numpy/blob/master/doc/DISTUTILS.rst.txt#id4

Comment (ocefpaf) [reply recipes/scikit-video/meta.yaml]: You can raise it upstream but my fuzzy memory tells me that is correct. Although things may have changed lately.

Comment (carlodri) [reply recipes/scikit-video/meta.yaml]: ok thanks, sorry for the many questions...

If you think numpy pinning is ok now you can merge and archive the old sk-video feedstock when CI passes.

Comment (ocefpaf) [reply recipes/scikit-video/meta.yaml]: > ok thanks, sorry for the many questions...

No problem.

> If you think numpy pinning is ok now you can merge and archive the old sk-video feedstock when CI passes.

Done!

**Comment by carlodri** [comment]
thanks for everything @ocefpaf!

**Pull request merged**